cmd/thanos/compact.go

```diff
@@ -16,11 +16,11 @@ import (
 	"github.com/improbable-eng/thanos/pkg/objstore/client"
 	"github.com/improbable-eng/thanos/pkg/runutil"
 	"github.com/oklog/run"
-	"github.com/opentracing/opentracing-go"
+	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
Why this change ?

Reply:
I'll have to take a look at the imports.. those are things Atom did, not sure why :D

Reply:
It's the same for vim-go I think it's some new default formating for packages, which import path != package name.

Reply:
So should I revert this or is it ok?

Reply:
it's ok